website/src/specs/yarn.lock

```diff
@@ -0,0 +1,3468 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
this isn't in the workspaces, so it gets its own yarn.lock and won't automatically be installed. To fix, add to root package.json#workspaces and remove yarn lock

Reply:
Yes it's much better and requires less clumsy scripts 👍

Astro can [only use stylesheets from the public directory (or absolute links) when using `<link>` tags](https://docs.astro.build/en/guides/styling/#load-a-static-stylesheet-via-link-tags) so I'm using [`prepare`](https://docs.npmjs.com/cli/v9/using-npm/scripts#prepare-and-prepublish) scripts to build the library, then copy the themes in `specs/public/`.